include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -312,6 +312,12 @@ bool print_transaction_table(Transaction & transaction) {
 
         ts_summary.add(tspkg.get_action());
         for (auto & replaced : tspkg.get_replaces()) {
+            // highlight incoming packages with epoch/version change
+            if (tspkg.get_package().get_epoch() != replaced.get_epoch() || tspkg.get_package().get_version() != replaced.get_version()) {
+                auto cl_evr = scols_line_get_cell(ln, COL_EVR);
+                scols_cell_set_color(cl_evr, "bold");
+            }
```

Comment:
Be honest, I think the change will make all versions bolt. We already uses 3 colors and additional playing with bolt will only makes other more important changes less visible. What about to remove the change from PR?

Reply:
Have you tested this change? It really doesn't print all versions in bold.
From a user / admin perspective highlighting changed versions is way more important than using colors for package actions.

Reply:
I tested but right now I am just after upgrade, therefore I do not see the right picture. Anyway I am not happy about the change because guessing importance of update according changed epoch and version is only guess with no significance. The proper source of information are change logs or advisories. When we will highlight it probably admins will think that we have a good source of information what is important and they will ignore relevant sources mentioned above.

May I ask you to split the PR into two or simply remove https://github.com/rpm-software-management/libdnf/pull/1405/commits/190bcae1914cbb44c9571783baebe685dfe24215?

The rest of PR is perfect. Thanks.

Reply:
We need to document meanings of colors / highlights in the transaction table anyway. And clear saying there that we highlight changed epoch/versions should prevent such confusion.
I quite like this change - it draws attention to packages where I might want to check changelogs.

Reply:
Ok, then we can keep it, but the note about documentation is a good point.